source/solvers/navier_stokes_base.cc

```diff
@@ -1424,13 +1437,13 @@ NavierStokesBase<dim, VectorType, DofsType>::postprocess_fd(bool firstIter)
             simulation_parameters.post_processing.flow_rate_output_name +
             ".dat";
           std::ofstream output(filename.c_str());
-          flow_rate_table.set_precision("time", 12);
+          flow_rate_table.set_precision("time", 4);
```

Comment:
I'm not sure this is a good idea. For the other variables we use 12 significant digits. Since this is always written to a data file, I think it's a much better idea to keep as many digits as possible. Using scientific notation is a good idea however.

Reply:
I think I misunderstood. Here is it the one you are printing on the screen or the one your are printing on the data file? For the screen on e I think 4 digit scientific is ok. For the data file one, I think it's not a good idea to print just 4 digits.

Reply:
For log outputted to the terminal you can also youse simulation_control->get_log_precision() which is meant exactly for that.

Reply:
Agreed, will keep the scientific notation but with 12 digits for the data file. For the screen display I'll keep 4 digits in scientific for better readability